### PR TITLE
E-785 feat(ui): tooltips for checkboxes

### PR DESCRIPTION
**Pull request by ifiri**

### CODE REVIEW

**Comment by ifiri** [comment]
@dudiq ready for review

**Comment by dudiq** [comment]
@lamartire ready for approve

File: packages/ui/stories/VCheckbox.stories.js [hunk 1]
```diff
@@ -161,6 +161,26 @@ storiesOf('VCheckbox/desktop', module)
...
+    data() {
+      return {
+        model: null,
+      };
+    },
+    template: `
+      <theme-provider>
+        <v-checkbox
+          v-model="model"
+          tooltip="I am tooltip"
```
Comment (lamartire) [inline packages/ui/stories/VCheckbox.stories.js:176]: Invalid property name

**Comment by ifiri** [comment]
@lamartire done

**Pull request merged**